### DISCUSSION TITLE
Wheelbase RJ12 Pedal pinout

### DISCUSSION THREAD

**Discussion by jarmenti** [FendtXerion3800/Fanatec-Connectivity#3] [Ideas]
I noticed you're missing the pedal input pinout on the wheelbase page..

pin 2 = handbrake signal
pin 3 = clutch
pin 4= brake
pin 5= gas

I just finished an aliexpress handbrake conversion to my CSW2.5 base, and wrote some arduino code to convert any of the cheap momentary switch/hall switch based shifters to work natively on the wheelbases. My goal was keeping console compatibility intact, and eliminate everything being a separate usb peripheral.

**Comment by FendtXerion3800** [comment]
Thx alot for sharing! Nice to hear that yu got your own devices working.
And Yeah the Pedal pinout is a wierd one. First of all to get on the same page. Is it possible that your pinout is mirrored to the pinout in the wiki? I find it quiet hard to get a consistent answer on that in the internet. In the German wikipedia site it is the same as I use but sometimes I encounter flipped pinouts. But non the less...

When I measured the voltages at each pin with no connection. When i remember correctly and based on my notes i got:

Pin 1: 0V (Presumably GND)
Pin 2: 0V
Pin 3: 0V
Pin 4: +3.3V
Pin 5:  0V
Pin 6: +5V

I think I tried hooking up some Potentiometers but could not get good results. Maybe i have to revisit the Pedal Connector.

Do you have some experience with DIY pedals for Fanatec or only the shifter and handbrake?

**Reply by jarmenti** [reply]
I've tested the wheel base pedal pinout with a mock home made pedal " set" which was just  a proto breadboard and potentiometers. This is how I figured out the pinout that I listed.   You are 100% correct with 1 and 6 pins being gnd and power.   

 I've also disassembled the CSL pedal set (non load cell), and they are as barebones as  it gets.... Just a 3 wire hall sensor in place of a potentiometer, wired straight between power gnd, and the according signal pin (gas brake clutch).

**Comment by madlajos** [comment]
Did you convert a Hall Sensor-based E-brake? I have trouble with my solution. It works fine if I connect it to my Clubsport V3 pedals (which is then connected to a PC via USB), but it does not work when plugged into a CSL DD directly.
Any help would be appreciated, as I'm starting to lose my mid over this!
![image](https://github.com/user-attachments/assets/54213a3f-d8d7-40e1-aab9-551a0668970f)
